### PR TITLE
{lang}[GCCcore/12.2.0] Perl v5.36.0, libreadline v8.2, ncurses v6.3, expat v2.4.9, ...

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
node3114.skitty.os - Linux RHEL 8.4, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/ade13c4162c31e81ac865b46abf26f0e for a full test report.

File: easybuild/easyconfigs/n/ncurses/ncurses-6.3-GCCcore-12.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+toolchain = {'name': 'GCCcore', 'version': '12.2.0'}
+toolchainopts = {'pic': True}
+
+source_urls = [GNU_SOURCE]
+sources = [SOURCE_TAR_GZ]
+checksums = ['97fc51ac2b085d4cde31ef4d2c3122c21abc217e9090a43a30fc5ec21684e059']
+
+builddependencies = [('binutils', '2.39')]
+
+local_common_configopts = "--with-shared --enable-overwrite --without-ada --enable-symlinks "
```
[the remark this answers is not in the document]
Comment (boegel) [reply easybuild/easyconfigs/n/ncurses/ncurses-6.3-GCCcore-12.2.0.eb:23]: Let's get #16064 merged first (which only touches `ncurses` easyconfigs using the `system` toolchain).

I would definitely tackle (recent) `ncurses` easyconfigs using a non-system toolchain in a separate PR anyway...

**Comment by boegel** [comment]
@boegelbot please test @ generoso
CORE_CNT=16

File: easybuild/easyconfigs/n/ncurses/ncurses-6.3-GCCcore-12.2.0.eb (hunk 1 above)
[the remark this answers is not in the document]
Comment (verdurin) [reply easybuild/easyconfigs/n/ncurses/ncurses-6.3-GCCcore-12.2.0.eb:23]: That's now merged.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen2
CORE_CNT=16

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**FAILED**
Build succeeded for 4 out of 6 (6 easyconfigs in total)
bdw-opensuse-154 - Linux openSUSE Leap 15.4, x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.6.15
See https://gist.github.com/58b1d1221ba71699b7516476641a8037 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**FAILED**
Build succeeded for 4 out of 6 (6 easyconfigs in total)
zen2-ubuntu-eb - Linux Ubuntu 22.04, x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.10.4
See https://gist.github.com/1ed2d2d8bee8776c6498b3482907469f for a full test report.

File: easybuild/easyconfigs/n/ncurses/ncurses-6.3-GCCcore-12.2.0.eb (hunk 1 above)
Comment (SebastianAchilles) [inline easybuild/easyconfigs/n/ncurses/ncurses-6.3-GCCcore-12.2.0.eb:23]: I am seeing the issue https://github.com/easybuilders/easybuild-easyconfigs/issues/16014 here as well. As in https://github.com/easybuilders/easybuild-easyconfigs/pull/16270 we need to add `--with-versioned-syms`
```suggestion
local_common_configopts = "--with-shared --enable-overwrite --without-ada --enable-symlinks --with-versioned-syms "
```

**Review by SebastianAchilles — CHANGES_REQUESTED** [review]

**Review by jfgrimm — COMMENTED** [review]
As decided in issue #16330, we have deprecated the use of `True` to signify a system-toolchain dependency (#16384), in favour of the more intuitive `SYSTEM` template constant. Due to the change in the test suite, please run `eb --sync-pr-with-develop 16204` and update the PR to use `SYSTEM` instead.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
bwd-rockylinux-90 - Linux Rocky Linux 9.0 (Blue Onyx), x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), Python 3.9.10
See https://gist.github.com/dfbe5f232dfab544312a1f44820e52b5 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
zen2-ubuntu-eb - Linux Ubuntu 22.04, x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.10.6
See https://gist.github.com/5d00a2c1947a3365ca2c3e9dcfec60bd for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
jrc0870.jureca - Linux Rocky Linux 8.6, AArch64, ARM UNKNOWN (graviton2), 2 x NVIDIA NVIDIA A100-PCIE-40GB, 515.65.01, Python 3.6.8
See https://gist.github.com/e9dc4039474085fd333ed9e16a3c281c for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Review by jfgrimm — APPROVED** [review]
LGTM

**Comment by jfgrimm** [comment]
Going in, thanks @boegel!

**Comment by branfosj** [comment]
Test report by @branfosj
**SUCCESS**
Build succeeded for 6 out of 6 (6 easyconfigs in total)
bear-pg0105u36b.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/564643d4f4eda15641808dd84b7d3286 for a full test report.

**Pull request merged**